src/tarballs/config.ts

```diff
@@ -12,6 +12,7 @@ export const TARGETS = [
   'win32-x64',
   'win32-x86',
   'darwin-x64',
+  'darwin-arm'
```

Comment:
I'm not sure that just adding this will work because if you look at https://nodejs.org/dist/v16.0.0/ the darwin arm executables are actually named `darwin-arm64` so this will either need to be changed or we might want to have some sort of conversion from different possible names people could use to `darwin-arm64` in the pack command (`darwin-m1`, `apple-silicone`, `m1`, etc....). Whatever you prefer.

Reply:
@RodEsp `arm` is translated to `armv7l` here: https://github.com/oclif/oclif/blob/master/src/tarballs/node.ts#L17

So `linux-arm` and `darwin-arm` are actually `linux-armv7l` and `darwin-armv7l`

We could add more explicit targets, e.g.
- `darwin-arm64`
- `darwin-armv7l`
- `linux-arm64`
- `linux-armv7l`

Either way we'd have to keep `linux-arm` for backwards compatibility 

I don't have a strong preference either way

Reply:
The `armv7l` architecture is for linux 32 bit though, not for darwin 64 bit. We need to differentiate between the two.
There is no such thing as `darwin-armv7l` afaik. Node only has `darwin-arm64` or `darwin-x64` distributables.

![image](https://user-images.githubusercontent.com/1084688/141352072-4b411daa-1e59-49d0-930d-57c8b262a5df.png)